src/bic/collection/form-records.js

```diff
@@ -33,14 +33,14 @@ define(function (require) {
 
             collection.reset();
 
-            nodes = data.evaluate('//' + formName, data);
+            nodes = data.evaluate('//' + formName, data, null, XPathResult.ANY_TYPE, null);
             node = nodes.iterateNext();
 
             parseNodes = function (key) {
               if (key.nodeName === 'id') {
-                parsed.id = key.innerHTML;
+                parsed.id = key.textContent;
```

Comment:
I wonder if this should be `key.textContent.trim()` ?

For example, would this work if the XML was a multi-line string with new-lines or other white-space in between the tags and the values (i.e. like a human-friendly XML file) ?